addons/account/views/account_invoice_view.xml

```diff
@@ -47,18 +47,35 @@
                     <field name="uom_id" groups="uom.group_uom"/>
                     <field name="price_unit"/>
                     <field name="discount" groups="base.group_no_one"/>
-                    <field name="price_subtotal"/>
+                    <field name="price_subtotal" groups="account.group_show_price_subtotal"/>
+                    <field name="price_total" groups="account.group_show_price_total"/>
                     <field name="currency_id" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+        <!--
+            TODO: This view uses "parent" and will not work standalone.
```

Comment:
I considered it a TODO is because it is a strange behaviour that we might want to improve, for example in the framework by ignoring parent and preventing the view from crashing when it it used standalone. Or by encoding this particular view differently.